kyuubi-rest-client/src/main/java/org/apache/kyuubi/client/BatchRestApi.java

```diff
@@ -0,0 +1,90 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.kyuubi.client;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import java.util.HashMap;
+import java.util.Map;
+import org.apache.kyuubi.client.api.v1.dto.Batch;
+import org.apache.kyuubi.client.api.v1.dto.BatchRequest;
+import org.apache.kyuubi.client.api.v1.dto.GetBatchesResponse;
+import org.apache.kyuubi.client.api.v1.dto.OperationLog;
+import org.apache.kyuubi.client.exception.KyuubiRestException;
+
+public class BatchRestApi {
+
+  private KyuubiRestClient client;
+
+  private static final String API_BASE_PATH = "batches";
+
+  public BatchRestApi(KyuubiRestClient client) {
+    this.client = client;
+  }
+
+  public Batch createBatch(BatchRequest request) throws KyuubiRestException {
+    String jsonBody = null;
+    try {
+      jsonBody = new ObjectMapper().writeValueAsString(request);
+    } catch (JsonProcessingException e) {
```

Comment:
catch Exception instead of JsonProcessingException, because there may RuntimeException occurs.